js-miniapp-sdk/src/index.ts

```diff
@@ -5,8 +5,11 @@
  */
 
 import { MiniApp } from './miniapp';
+import { AdTypes } from './types/adTypes';
+import { InterstitialAdResponse } from './types/responseTypes/interstitial';
+import { RewardedAdResponse } from './types/responseTypes/rewarded';
 
 /** @internal */
 const miniAppInstance = new MiniApp();
 
-export = miniAppInstance;
+export { miniAppInstance, AdTypes, InterstitialAdResponse, RewardedAdResponse };
```

Comment:
I don't think this is correct because now to access the SDK methods you'd have to use `import { miniAppInstance } from 'js-miniapp-sdk'`. How about this instead?

```suggestion
export default miniAppInstance;
export { AdTypes, InterstitialAdResponse, RewardedAdResponse };
```

Then it can be used like `import miniApp, { AdTypes } from 'js-miniapp-sdk'`.

Note that I think our TSLint rules currently don't allow `default` exports, but I don't think there's a good reason for that. So you should be able to change this by adding to `tslint.json`:
```
    "rules": {
        "no-default-export": false
    }
```